pkgs/servers/krill/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, stdenv
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "krill";
+  version = "0.9.4";
+
+  src = fetchFromGitHub {
+    owner = "NLnetLabs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256:1w5s4vmr126k5lnxsv3hfxx1yz2c1gp3hmwfz5jn4lar78r5kidy";
+  };
+
+  cargoSha256 = "sha256:0ksh68giq3yd8k7vynxdl13ccysl6i5ag2nz9nknbnrvpv201c88";
+
+ buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ Security ];
```

Comment:
```suggestion
 buildInputs = [ openssl ] ++ lib.optional stdenv.isDarwin Security;
```

Reply:
Thanks :). I have redone the indentation and removed one space EOL. Was that what you meant with the formatting?

I've also renamed the maintainer commit.